### PR TITLE
KEYCLOAK-17861 email notification for add/remove group

### PR DESCRIPTION
**Pull request by cgeorgilakis**
https://github.com/keycloak/keycloak/pull/7965

### CODE REVIEW

File: server_admin/topics/groups.adoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ image:{project_images}/user-groups.png[]
 Select a group from the `Available Groups` tree and hit the `join` button to add the user to a group.  Vice versa to remove a group.
 Here we've added the user _Jim_ to the _North America_ sales group.  If you go back to the detail page for that group and
 select the `Membership` tab, _Jim_ is now displayed there.
+User is notified via email every time he is added to/ removed from a group.
```
Comment (andymunro) [inline server_admin/topics/groups.adoc]: Avoid latin words such as "via" and gender-based words such as "he"

Suggested:  The user is notified by email when that user is added to or removed from a group.

**Review by andymunro — CHANGES_REQUESTED** [review]
One change suggested.

**Comment by andymunro** [comment]
@cgeorgilakis You will need to use this command to fix the verification build
git pull upstream main

**Comment by cgeorgilakis** [comment]
I have rebased from keycloak main and have applied your proposal to new proc-managing-groups.adoc

**Review by andymunro — APPROVED** [review]

**Comment by ahus1** [comment]
The documentation has been moved to [the main Keycloak repository](https://github.com/keycloak/keycloak/tree/main/docs/documentation).

Please include these documentation changes into the main pull request.

Thanks!

**Pull request closed without merging**